int16 zone1
int16 zone2
int16 zone3


